offer-manager-domain/src/main/groovy/life/qbic/business/offers/QuantityDiscount.groovy

```diff
@@ -12,17 +12,17 @@ import java.util.function.BiFunction
 class QuantityDiscount implements BiFunction<Integer, Double, Double> {
 
     /**
-     * Applies a quantity discount to the discountable price depending on the number of samples.
+     * Computes a quantity discount of the full price depending on the number of samples.
      * It is defined for all positive natural numbers greater than 0
      *
      * @param sampleCount the number of samples
-     * @param discountablePrice the price that should be discounted
-     * @return the function the discountablePrice with discounts applied to it
+     * @param fullPrice the price that should be discounted, before discounts are apploed
+     * @return the size of the discount
```

Comment:
```suggestion
     * @return the amount to be discounted
```

Reply:
this sounds really bad in my opinion 😅 
What about "the discount amount" ?